### ISSUE TITLE
No converter for [class io.tolgee.exceptions.ErrorResponseBody] with preset Content-Type 'text/event-stream;charset=UTF-8'

### ISSUE THREAD

**Issue by wfjsw** [tolgee/tolgee-platform#1754]
Version: 3.16.0

```
Jun 13 09:27:14 ank java[2795431]: org.springframework.http.converter.HttpMessageNotWritableException: No converter for [class io.tolgee.exceptions.ErrorResponseBody] with preset Content-Type 'text/event-stream;charset=UTF-8'
Jun 13 09:27:14 ank java[2795431]:         at org.springframework.web.servlet.mvc.method.annotation.AbstractMessageConverterMethodProcessor.writeWithMessageConverters(AbstractMessageConverterMethodProcessor.java:312) ~[spring-webmvc-5.3.27.jar!>
Jun 13 09:27:14 ank java[2795431]:         at org.springframework.web.servlet.mvc.method.annotation.HttpEntityMethodProcessor.handleReturnValue(HttpEntityMethodProcessor.java:219) ~[spring-webmvc-5.3.27.jar!/:5.3.27]
Jun 13 09:27:14 ank java[2795431]:         at org.springframework.web.method.support.HandlerMethodReturnValueHandlerComposite.handleReturnValue(HandlerMethodReturnValueHandlerComposite.java:78) ~[spring-web-5.3.27.jar!/:5.3.27]
Jun 13 09:27:14 ank java[2795431]:         at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:135) ~[spring-webmvc-5.3.27.jar!/:5.3.27]
Jun 13 09:27:14 ank java[2795431]:         at org.springframework.web.servlet.mvc.method.annotation.ExceptionHandlerExceptionResolver.doResolveHandlerMethodException(ExceptionHandlerExceptionResolver.java:428) ~[spring-webmvc-5.3.27.jar!/:5.3.2>
Jun 13 09:27:14 ank java[2795431]:         at org.springframework.web.servlet.handler.AbstractHandlerMethodExceptionResolver.doResolveException(AbstractHandlerMethodExceptionResolver.java:75) ~[spring-webmvc-5.3.27.jar!/:5.3.27]
Jun 13 09:27:14 ank java[2795431]:         at org.springframework.web.servlet.handler.AbstractHandlerExceptionResolver.resolveException(AbstractHandlerExceptionResolver.java:142) ~[spring-webmvc-5.3.27.jar!/:5.3.27]
Jun 13 09:27:14 ank java[2795431]:         at org.springframework.web.servlet.handler.HandlerExceptionResolverComposite.resolveException(HandlerExceptionResolverComposite.java:80) ~[spring-webmvc-5.3.27.jar!/:5.3.27]
Jun 13 09:27:14 ank java[2795431]:         at org.springframework.web.servlet.DispatcherServlet.processHandlerException(DispatcherServlet.java:1332) ~[spring-webmvc-5.3.27.jar!/:5.3.27]
Jun 13 09:27:14 ank java[2795431]:         at org.springframework.web.servlet.DispatcherServlet.processDispatchResult(DispatcherServlet.java:1143) ~[spring-webmvc-5.3.27.jar!/:5.3.27]
Jun 13 09:27:14 ank java[2795431]:         at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1089) ~[spring-webmvc-5.3.27.jar!/:5.3.27]
Jun 13 09:27:14 ank java[2795431]:         at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:965) ~[spring-webmvc-5.3.27.jar!/:5.3.27]
Jun 13 09:27:14 ank java[2795431]:         at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1006) ~[spring-webmvc-5.3.27.jar!/:5.3.27]
Jun 13 09:27:14 ank java[2795431]:         at org.springframework.web.servlet.FrameworkServlet.doGet(FrameworkServlet.java:898) ~[spring-webmvc-5.3.27.jar!/:5.3.27]
Jun 13 09:27:14 ank java[2795431]:         at javax.servlet.http.HttpServlet.service(HttpServlet.java:529) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 09:27:14 ank java[2795431]:         at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:883) ~[spring-webmvc-5.3.27.jar!/:5.3.27]
Jun 13 09:27:14 ank java[2795431]:         at javax.servlet.http.HttpServlet.service(HttpServlet.java:623) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 09:27:14 ank java[2795431]:         at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:209) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 09:27:14 ank java[2795431]:         at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 09:27:14 ank java[2795431]:         at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:102) ~[spring-web-5.3.27.jar!/:5.3.27]
Jun 13 09:27:14 ank java[2795431]:         at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 09:27:14 ank java[2795431]:         at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 09:27:14 ank java[2795431]:         at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:102) ~[spring-web-5.3.27.jar!/:5.3.27]
Jun 13 09:27:14 ank java[2795431]:         at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 09:27:14 ank java[2795431]:         at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 09:27:14 ank java[2795431]:         at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:102) ~[spring-web-5.3.27.jar!/:5.3.27]
Jun 13 09:27:14 ank java[2795431]:         at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 09:27:14 ank java[2795431]:         at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 09:27:14 ank java[2795431]:         at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:102) ~[spring-web-5.3.27.jar!/:5.3.27]
Jun 13 09:27:14 ank java[2795431]:         at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 09:27:14 ank java[2795431]:         at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 09:27:14 ank java[2795431]:         at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:102) ~[spring-web-5.3.27.jar!/:5.3.27]
Jun 13 09:27:14 ank java[2795431]:         at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 09:27:14 ank java[2795431]:         at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 09:27:14 ank java[2795431]:         at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:102) ~[spring-web-5.3.27.jar!/:5.3.27]
Jun 13 09:27:14 ank java[2795431]:         at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 09:27:14 ank java[2795431]:         at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 09:27:14 ank java[2795431]:         at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:102) ~[spring-web-5.3.27.jar!/:5.3.27]
Jun 13 09:27:14 ank java[2795431]:         at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 09:27:14 ank java[2795431]:         at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 09:27:14 ank java[2795431]:         at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:102) ~[spring-web-5.3.27.jar!/:5.3.27]
Jun 13 09:27:14 ank java[2795431]:         at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 09:27:14 ank java[2795431]:         at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 09:27:14 ank java[2795431]:         at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:100) ~[spring-web-5.3.27.jar!/:5.3.27]
Jun 13 09:27:14 ank java[2795431]:         at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117) ~[spring-web-5.3.27.jar!/:5.3.27]
Jun 13 09:27:14 ank java[2795431]:         at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 09:27:14 ank java[2795431]:         at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 09:27:14 ank java[2795431]:         at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:102) ~[spring-web-5.3.27.jar!/:5.3.27]
Jun 13 09:27:14 ank java[2795431]:         at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 09:27:14 ank java[2795431]:         at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[tomcat-embed-core-9.0.74.jar!/:na]
...
```

**Comment by wfjsw** [comment]
```
Jun 13 11:38:39 ank java[3185809]: 2023-06-13 11:38:39.543  WARN 3185809 --- [io-21062-exec-1] .m.m.a.ExceptionHandlerExceptionResolver : Failure in @ExceptionHandler io.tolgee.ExceptionHandlers#handleOtherExceptions(Throwable)
Jun 13 11:38:39 ank java[3185809]: org.springframework.http.converter.HttpMessageNotWritableException: No converter for [class io.tolgee.exceptions.ErrorResponseBody] with preset Content-Type 'application/javascript;charset=UTF-8'
Jun 13 11:38:39 ank java[3185809]:         at org.springframework.web.servlet.mvc.method.annotation.AbstractMessageConverterMethodProcessor.writeWithMessageConverters(AbstractMessageConverterMethodProcessor.java:312) ~[spring-webmvc-5.3.27.jar!/:5.3.27]
Jun 13 11:38:39 ank java[3185809]:         at org.springframework.web.servlet.mvc.method.annotation.HttpEntityMethodProcessor.handleReturnValue(HttpEntityMethodProcessor.java:219) ~[spring-webmvc-5.3.27.jar!/:5.3.27]
Jun 13 11:38:39 ank java[3185809]:         at org.springframework.web.method.support.HandlerMethodReturnValueHandlerComposite.handleReturnValue(HandlerMethodReturnValueHandlerComposite.java:78) ~[spring-web-5.3.27.jar!/:5.3.27]
Jun 13 11:38:39 ank java[3185809]:         at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:135) ~[spring-webmvc-5.3.27.jar!/:5.3.27]
Jun 13 11:38:39 ank java[3185809]:         at org.springframework.web.servlet.mvc.method.annotation.ExceptionHandlerExceptionResolver.doResolveHandlerMethodException(ExceptionHandlerExceptionResolver.java:428) ~[spring-webmvc-5.3.27.jar!/:5.3.27]
Jun 13 11:38:39 ank java[3185809]:         at org.springframework.web.servlet.handler.AbstractHandlerMethodExceptionResolver.doResolveException(AbstractHandlerMethodExceptionResolver.java:75) ~[spring-webmvc-5.3.27.jar!/:5.3.27]
Jun 13 11:38:39 ank java[3185809]:         at org.springframework.web.servlet.handler.AbstractHandlerExceptionResolver.resolveException(AbstractHandlerExceptionResolver.java:142) ~[spring-webmvc-5.3.27.jar!/:5.3.27]
Jun 13 11:38:39 ank java[3185809]:         at org.springframework.web.servlet.handler.HandlerExceptionResolverComposite.resolveException(HandlerExceptionResolverComposite.java:80) ~[spring-webmvc-5.3.27.jar!/:5.3.27]
Jun 13 11:38:39 ank java[3185809]:         at org.springframework.web.servlet.DispatcherServlet.processHandlerException(DispatcherServlet.java:1332) ~[spring-webmvc-5.3.27.jar!/:5.3.27]
Jun 13 11:38:39 ank java[3185809]:         at org.springframework.web.servlet.DispatcherServlet.processDispatchResult(DispatcherServlet.java:1143) ~[spring-webmvc-5.3.27.jar!/:5.3.27]
Jun 13 11:38:39 ank java[3185809]:         at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1089) ~[spring-webmvc-5.3.27.jar!/:5.3.27]
Jun 13 11:38:39 ank java[3185809]:         at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:965) ~[spring-webmvc-5.3.27.jar!/:5.3.27]
Jun 13 11:38:39 ank java[3185809]:         at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1006) ~[spring-webmvc-5.3.27.jar!/:5.3.27]
Jun 13 11:38:39 ank java[3185809]:         at org.springframework.web.servlet.FrameworkServlet.doPost(FrameworkServlet.java:909) ~[spring-webmvc-5.3.27.jar!/:5.3.27]
Jun 13 11:38:39 ank java[3185809]:         at javax.servlet.http.HttpServlet.service(HttpServlet.java:555) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 11:38:39 ank java[3185809]:         at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:883) ~[spring-webmvc-5.3.27.jar!/:5.3.27]
Jun 13 11:38:39 ank java[3185809]:         at javax.servlet.http.HttpServlet.service(HttpServlet.java:623) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 11:38:39 ank java[3185809]:         at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:209) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 11:38:39 ank java[3185809]:         at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 11:38:39 ank java[3185809]:         at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:102) ~[spring-web-5.3.27.jar!/:5.3.27]
Jun 13 11:38:39 ank java[3185809]:         at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 11:38:39 ank java[3185809]:         at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 11:38:39 ank java[3185809]:         at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:102) ~[spring-web-5.3.27.jar!/:5.3.27]
Jun 13 11:38:39 ank java[3185809]:         at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 11:38:39 ank java[3185809]:         at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 11:38:39 ank java[3185809]:         at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:102) ~[spring-web-5.3.27.jar!/:5.3.27]
Jun 13 11:38:39 ank java[3185809]:         at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 11:38:39 ank java[3185809]:         at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 11:38:39 ank java[3185809]:         at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:102) ~[spring-web-5.3.27.jar!/:5.3.27]
Jun 13 11:38:39 ank java[3185809]:         at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 11:38:39 ank java[3185809]:         at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 11:38:39 ank java[3185809]:         at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:102) ~[spring-web-5.3.27.jar!/:5.3.27]
Jun 13 11:38:39 ank java[3185809]:         at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 11:38:39 ank java[3185809]:         at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 11:38:39 ank java[3185809]:         at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:102) ~[spring-web-5.3.27.jar!/:5.3.27]
Jun 13 11:38:39 ank java[3185809]:         at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 11:38:39 ank java[3185809]:         at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 11:38:39 ank java[3185809]:         at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:102) ~[spring-web-5.3.27.jar!/:5.3.27]
Jun 13 11:38:39 ank java[3185809]:         at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 11:38:39 ank java[3185809]:         at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 11:38:39 ank java[3185809]:         at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:102) ~[spring-web-5.3.27.jar!/:5.3.27]
Jun 13 11:38:39 ank java[3185809]:         at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 11:38:39 ank java[3185809]:         at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 11:38:39 ank java[3185809]:         at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:100) ~[spring-web-5.3.27.jar!/:5.3.27]
Jun 13 11:38:39 ank java[3185809]:         at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117) ~[spring-web-5.3.27.jar!/:5.3.27]
Jun 13 11:38:39 ank java[3185809]:         at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 11:38:39 ank java[3185809]:         at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 11:38:39 ank java[3185809]:         at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:102) ~[spring-web-5.3.27.jar!/:5.3.27]
Jun 13 11:38:39 ank java[3185809]:         at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 11:38:39 ank java[3185809]:         at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[tomcat-embed-core-9.0.74.jar!/:na]
Jun 13 11:38:39 ank java[3185809]:         at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:102) ~[spring-web-5.3.27.jar!/:5.3.27]
```

**Comment by JanCizmar** [comment]
Well, this doesn't look like the original error. Is there anything else in the logs? Does this happen as a result of some operation?

**Comment by wfjsw** [comment]
These are the only log in the output file. They occurred when 2 teammate and me was browsing the translation history and doing review, then tolgee go unresponsive and I found this in log file. The first one is gone when I find out that I did not allow websocket connection on the `/websocket` endpoint (but I do run socket.io on a separate port and map it to `/aio`... seems it is not used). 

Here is the full log: 
[tolgee.log](https://github.com/tolgee/tolgee-platform/files/11734879/tolgee.log)

**Comment by JanCizmar** [comment]
Tolgee doesn't use socket.io anymore. It uses native Spring Websocket support (https://docs.spring.io/spring-integration/reference/html/web-sockets.html). 

According to the logs it looks like there is something broken with the handshake.

```
Jun 13 10:58:19 ank java[3185809]: 2023-06-13 10:58:19.623 ERROR 3185809 --- [io-21062-exec-7] o.s.w.s.s.s.DefaultHandshakeHandler      : "Handshake failed due to invalid Upgrade header: null"
```

There is a thread about this on Stack Overflow: https://stackoverflow.com/questions/31211919/spring-websocket-handshake-failed-due-to-invalid-upgrade-header-null

>  The first one is gone when I find out that I did not allow websocket connection on the /websocket endpoint

So does it work now?

**Comment by wfjsw** [comment]
I did add the relevant directives to the nginx, though later I find out that I did not enable the Websocket option on Cloudflare.

> So does it work now?

No. The error is substituted by a new one in https://github.com/tolgee/tolgee-platform/issues/1754#issuecomment-1589134947 and the attached log file. 

Now I turned the option on at Cloudflare, but the connection has not been not established yet. I'll wait to see if anything else happens.
